QEfficient/transformers/models/modeling_auto.py

```diff
@@ -174,8 +174,8 @@ def export(self, model_card_name: Optional[str] = None) -> str:
         Returns:
             :str: Path of the generated ``ONNX`` graph.
         """
-        assert self.is_transformed, "Please first run transform on the QEFFAutoModelForCausalLM object"
-
+        if not self.is_transformed:
```

Comment:
remove, as its already vhecking inside export func. This is duplicate

Reply:
Removed